subsys/bluetooth/shell/has.c

```diff
@@ -0,0 +1,155 @@
+/** @file
+ *  @brief Bluetooth Hearing Access Service (HAS) server shell.
+ *
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/types.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/audio/has.h>
+#include <shell/shell.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "bt.h"
+
+static struct bt_has *has;
+
+static const uint8_t g_universal_idx = 1;
+static const uint8_t g_outdoor_idx = 5;
+static const uint8_t g_noisy_idx = 8;
+static const uint8_t g_office_idx = 22;
+
+static int set_active_preset_cb(struct bt_has *has, uint8_t index, bool sync)
+{
+	int err;
+
+	shell_print(ctx_shell, "Set active preset index 0x%02x sync %d", index, sync);
+
+	err = bt_has_preset_active_set(has, index);
+	if (err < 0) {
+		shell_error(ctx_shell, "Set active failed (err %d)", err);
+	}
+	return err;
+}
+
+struct bt_has_preset_ops preset_ops = {
+	.active_set = set_active_preset_cb,
+};
+
+static int cmd_has(const struct shell *sh, size_t argc, char **argv)
+{
+	if (argc > 1) {
+		shell_error(sh, "%s unknown parameter: %s", argv[0], argv[1]);
+	} else {
+		shell_error(sh, "%s Missing subcomand", argv[0]);
+	}
+
+	return -ENOEXEC;
+}
+
+static int cmd_has_init(const struct shell *sh, size_t argc, char **argv)
+{
+	int result;
+	struct bt_has_register_param register_param = {
+		.preset_param = {
+			{.idx = g_universal_idx,
+			.properties = BT_HAS_PROP_WRITABLE | BT_HAS_PROP_AVAILABLE,
+			.name = "Universal"},
```

Comment:
```suggestion
			{
				.idx = g_universal_idx,
				.properties = BT_HAS_PROP_WRITABLE | BT_HAS_PROP_AVAILABLE,
				.name = "Universal"
			},
```
to keep the same style of `{`